android/src/main/java/com/github/yamill/orientation/listeners/OrientationConfigListener.kt

```diff
@@ -12,21 +12,19 @@ import com.facebook.react.bridge.LifecycleEventListener
 import com.facebook.react.bridge.ReactContext
 import com.facebook.react.common.ReactConstants
 import com.facebook.react.modules.core.DeviceEventManagerModule
+import com.github.yamill.orientation.util.OrientationUtil
 
 class OrientationConfigListener internal constructor(
-    reactContext: ReactContext,
+    private val reactContext: ReactContext,
     private val onGetCurrentActivity: () -> Activity?,
 ) : LifecycleEventListener {
 
     private val receiver = object : BroadcastReceiver() {
 
         override fun onReceive(context: Context, intent: Intent) {
             val newConfig = intent.getParcelableExtra<Configuration>(INTENT_VALUE_KEY)!!
-            val orientationValue = if (newConfig.orientation == 1) {
```

Comment:
The old logic only emitted changes for PORTRAIT and LANDSCAPE but not UNKNOWN. Is it intentional that the new code can also emit changes for UNKNOWN?

Reply:
However, I just looked at the upstream library (Java implementation), and it has the same behavior as the "old" logic.  In the spirit of preserving functionality (and matching the upstream library behavior), I will revert my change here to adhere to the previous logic.